src/style.css

```diff
@@ -1,3 +1,78 @@
-body {
+body,
+html {
   margin: 0;
+  padding: 0;
+  font-family: 'Roboto', sans-serif;
+}
+
+.heading {
+  text-align: center;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  background-color: #f8f8f8;
+}
+
+.logo {
+  margin: 0 40px;
+  width: 40px;
+  height: 40px;
+}
+
+nav {
+  flex-grow: 1;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.nav__list li {
+  margin: 0 15px;
+}
+
+.nav__list {
+  list-style: none;
+  margin: 0 60px;
+  padding: 0;
+  justify-content: flex-end;
+  display: flex;
+  font-size: 14px;
+  font-weight: bold;
+  align-items: center;
+}
+
+.nav__link {
+  line-height: 60px;
+  text-align: left;
+  color: black;
+  text-transform: uppercase;
+  text-decoration: none;
+  font-weight: bold;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
```

Comment:
Create a variable for repeated color and use it everywhere

Reply:
+